.github/workflows/js-ci.yml

```diff
@@ -0,0 +1,57 @@
+name: '[js] CI'
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+    branches: [master]
+    paths:
+      - 'js/**'
+      - 'package.json'
+  workflow_dispatch:
```

Comment:
What's the point in this trigger if there is nothing is executed when triggered manually?